examples/vision/deeplabv3_plus.py

```diff
@@ -1,12 +1,12 @@
 """
-Title: Multiclass semantic segmentation using DeepLabV3+
-Author: [Soumik Rakshit](http://github.com/soumik12345)
-Date created: 2021/08/31
-Last modified: 2021/09/1
-Description: Implement DeepLabV3+ architecture for Multi-class Semantic Segmentation.
+Title: FILLME
+Author: FILLME
+Date created: FILLME
+Last modified: FILLME
+Description: FILLME
 """
 """
-## Introduction
+ Introduction
```

Comment:
E.g. formatting changes, etc.

Reply:
Removed unnecessary formatting changes introduced in notebook conversion.